css/main.scss

```diff
@@ -2051,7 +2049,7 @@ button {
   flex-direction: column;
 }
 .doc-content {
-  padding: 105px 0 80px;
+  padding: 105px 120px 80px;
```

Comment:
120px was there before, how come it is 0 again? Did you merged with latest changes?